src/Files.App/ResourceDictionaries/ComboBoxStyle.xaml

```diff
@@ -0,0 +1,9 @@
+<!--  Copyright (c) 2024 Files Community. Licensed under the MIT License. See the LICENSE.  -->
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+	<Style x:Key="ComboBoxStyle" TargetType="ComboBox">
+		<Setter Property="CornerRadius" Value="4" />
```

Comment:
Use StaticResource at least.
And I think this isn’t needed because this is the default value.

Reply:
I thought it was default too, but it didn't show up when I tested. So I manually set it.

Reply:
If you'd like this to be default, you can name it as 'DefaultComboboxStyle'

Reply:
Ah you didn't set a style to be based on. You should add BasedOn="static resource DefaultComboboxStyle"(not correct grammar tho).

Reply:
Fixed!